index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="description" content="About Git">
+    <meta name="author" content="Mohammed Aslam">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Git</title>
+    <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>About Git</h1>
+    </header>
+
+<main>
+        <section class="grid-container">
+            <article class="article">
+                <img class="centered" src="git.avif" alt="Description of Image 1" width = "640" height = "380">
+                <h2>What is Git?</h2>
+                <p>Git basically takes a picture of what all your files look like at that moment and stores a reference to that snapshot.</p>
+               	<a href="https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F" target = "_blank">Read more to find out what Git is</a>
+            </article>
+            <article class="article">
+                <img src="developer.webp" alt="Description of Image 2" width = "640" height = "380">
+                <h2>Why do developers need Git?</h2>
+                <p>Git is the most commonly used version control system. Git tracks the changes you make to files, so you have a record of what has been done, and you can revert to specific versions should you ever need to.</p>
+                <a href = "https://git-scm.com/about/branching-and-merging" target = "_blank">Read More to find out why developers need Git</a>
+            </article>
+            <article class="article">
+                <img src="gitbranch.webp" alt="Description of Image 3" width = "640" height = "380">
```

Comment:
what does "Description of Image 3" mean?

Or 1 or 2? What should go here, and why did you write this instead?

Reply:
These image description name have been updates stating what the image represent. I saved them as Description of Image 3 which of course isn't meaningful. This has been amended in my index.html.